sdk/cosmos/azure-cosmos-encryption/pom.xml

```diff
@@ -41,7 +41,8 @@ Licensed under the MIT License.
     <!-- CosmosSkip - This is not a module we want/expect external customers to consume. Skip breaking API checks. -->
     <!-- This can only be enabled once we release GA, as it needs a stable version to check for breaking changes. -->
     <revapi.skip>true</revapi.skip>
-
+    <!-- We are adding code for AAP jars, once AAP jar is GAed to maven, we will remove spotbugs.skip-->
+    <spotbugs.skip>true</spotbugs.skip>
```

Comment:
Will this disable spot bug check for every class under encryption? 
is there a way to only skip spot bug check for classes under com.azure.cosmos.encryption.mdesrc?

Reply:
Done, added exclusion for aap source code in spotbug exclusion file